Docs: Added Badges to readme

NegiAkash890 opened the pull request:
### Description

This makes changes to the main README of the repository ( anitab-org /open-source-programs-web ) . The task was to add badges at the top of README .

Fixes #84 

### Type of Change:

- Documentation

**Code/Quality Assurance Only**

- This change requires a documentation update (software upgrade on readme file)

### How Has This Been Tested?
Screenshot of testing 

![Screenshot from 2021-02-23 09-56-17](https://user-images.githubusercontent.com/55234838/108802296-f3885900-75bd-11eb-8862-573710f2512a.png)



### Checklist:


- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials
- [X] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation



**Code/Quality Assurance Only**
- [X] My changes generate no new warnings

NegiAkash890 committed:
Updated - README.md

This adds badges to  README.md  .

Aaishpra reviewed (commented):
Please try to follow the PR template and use a better PR title (may be use the same title as the linked issue)

NegiAkash890 commented:
Sure . I will make sure next time that I follow PR template as well as use a better PR title :) . Let me know if there can any other changes that needs to be done to the above PR .

NegiAkash890 commented:
Sure . I will make sure next time that I follow PR template as well as use a better PR title :) . Also , please let me know if there are any other changes remaining that needs to be done to the above PR .

Aaishpra commented on README.md:
```diff
@@ -1,3 +1,11 @@
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/235478-anitab-org.2Egithub.2Eio)
```
The link attached is wrong here (takes us to anitab-org website chat)please use this link https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs

Aaishpra commented:
You can edit the PR description and the PR title by clicking on the three dots at the side. Ping me if you have any problem doing that. (The thing is the correct PR description and checklist makes it easier for reviewers to review the PR)

NegiAkash890 committed:
docs : Fixed link of Badge "join chat" 

This commit was made to address the issue #84 i.e. " docs : Add Badges to readme " .

Aaishpra reviewed (dismissed):
Lgtm 🎉

codesankalp commented on README.md:
```diff
@@ -1,3 +1,11 @@
+[![project chat](https://img.shields.io/badge/zulip-join_chat-brightgreen.svg)](https://anitab-org.zulipchat.com/#narrow/stream/237907-open-source-progs)
+[![License: GPL v3](https://img.shields.io/badge/License-GPL%20v3-blue.svg)](http://www.gnu.org/licenses/gpl-3.0)
+![Open Source Love](https://badges.frapsoft.com/os/v1/open-source.svg?v=102)
```
```suggestion
![Open Source Love](https://badges.frapsoft.com/os/v3/open-source.svg?v=103)
```

codesankalp reviewed (changes requested):
![image](https://user-images.githubusercontent.com/56037184/108599807-5e1c7780-73b9-11eb-9c19-6e2542f381de.png)

Can you do the suggested changes so that it will show in firefox?

NegiAkash890 committed:
docs : Fixed badge visibitity issue .

This commit is made to fix the visibility of  "open source" badge in the README.md of this repository .

Co-authored-by: Sankalp <sankalp123427@gmail.com>

codesankalp reviewed (dismissed):
Thanks @NegiAkash890 for your contribution 🎉

isabelcosta commented:
@NegiAkash890 can you please follow @Aaishpra comment, and update the PR description and title? You can edit this way:

![image](https://user-images.githubusercontent.com/11148726/108641170-c6d92200-7495-11eb-9b36-4d37aff7265e.png)

Here's the PR template for you to follow https://github.com/anitab-org/open-source-programs-web/blob/develop/.github/PULL_REQUEST_TEMPLATE.md

Rahulm2310 reviewed (commented):
@NegiAkash890 can you use the `shields` badge for `Open Source Love` badge as well as it seems to be little inconsistent with all other badges. 
![image](https://user-images.githubusercontent.com/54268438/108754282-be531b00-756b-11eb-8763-8741fe1eb3d4.png)

How about [this](https://img.shields.io/badge/Open%20%E2%9D%A4%EF%B8%8FSource-blue)

Rahulm2310 commented:
@NegiAkash890 Can you please add a screenshot of the changes under the `How has this been tested` section so that a reviewer knows what changes this PR makes. Thanks

NegiAkash890 committed:
Updated - README.md

This commit makes changes in the "open souce love"  badge of the README.md

Aaishpra reviewed (commented):
@NegiAkash890 just change PR title to `Docs: Added Badges to readme` other than this everything else looks good.

codesankalp reviewed (dismissed):
LGTM :tada: @NegiAkash890 
Tagging @isabelcosta  for final review.

Aaishpra reviewed (dismissed):
Great work
Ready to merge🚀

isabelcosta commented on README.md:
```diff
@@ -1,3 +1,11 @@
+![Open Source Love](https://img.shields.io/badge/Open%20%E2%9D%A4%EF%B8%8FSource-blue)
+[![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/open-source-programs-web)
+![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
+[![GitHub forks](https://img.shields.io/github/forks/anitab-org/open-source-programs-web?style=social)](https://github.com/anitab-org/open-source-programs-web/network/members)
+[![GitHub stars](https://img.shields.io/github/stars/anitab-org/open-source-programs-web?style=social)](https://github.com/anitab-org/open-source-programs-web/stargazers)
+
```
These are redundant since this is present in the GitHub interface. Could you remove them please?

NegiAkash890 committed:
docs : Removed redundant badges . 

This commit removes "Forks" and "Stars" badges from the README.md of the repository .
